index.html

```diff
@@ -13,11 +13,11 @@
 
 <body>
     <h1>Hangman Demo</h1>
-    <div class='failure'></div>
-    <div class='word'></div>
-    <div>Wrong letters guessed: <span class='wrong-letters'></span></div>
-    <div>Number of attemptes remaining: <span class='attempts-remaining'></span></div>
-    <div>Number of wins: <span class='wins'>0</span></div>
```

Comment:
Adds spacing between text for easier readability; fixes typo **attemptes** to **attempts**